lib/main_common.pm

```diff
@@ -1705,7 +1705,6 @@ sub load_extra_tests_sdk {
 
 sub load_extra_tests_docker {
     my ($image_names, $stable_names) = get_suse_container_urls();
-    return unless @$image_names;
```

Comment:
Plz give me a node why you are removing it.
For me it only has effect for `SLE <=12-SP2`, `LEAP >15 @ ppc64le` and `SLE >=12-SP3 <15 @ aarch64`. But I don't see the point why we need to remove it.

Reply:
this was my first question , but than I realize that I can not understand why this variables needed at all . Do you know ?

Reply:
For public cloud, `@$image_names` is empty so the docker tests were skipped. For SLE `suse_container_urls.pm` contains only the entries from the internal registries and we skip those, thus it's empty.

Following @pdostal comment, I now skip `docker_image.pm`, if those variables empty.